README.md

```diff
@@ -14,43 +14,25 @@ Developer settings is not displayed in production mode. To enable this feature i
 ### How to install module
 
 #### Using Git
-1. Clone repository: `git clone https://github.com/adifucan/m2-baler.git`
-2. Rename cloned `m2-baler` directory into `Baler` and place this directory into `app/code/Magento` directory
-3. Run `bin/magento setup:upgrade`
+
+```bash
+mkdir app/code/Magento
+git clone https://github.com/adifucan/m2-baler.git app/code/Magento/Baler
```

Comment:
Correct url is https://github.com/adifucan/m2-baler.git but I'll fix it

Reply:
Thanks